src/driver/oracle/OracleQueryRunner.ts

```diff
@@ -1429,7 +1432,7 @@ export class OracleQueryRunner extends BaseQueryRunner implements QueryRunner {
         if (column.isNullable !== true && !column.isGenerated) // NOT NULL is not supported with GENERATED
             c += " NOT NULL";
         if (column.isGenerated === true && column.generationStrategy === "increment")
-            c += " GENERATED ALWAYS AS IDENTITY";
+            c += " GENERATED BY DEFAULT AS IDENTITY";
```

Comment:
https://www.oracletutorial.com/oracle-basics/oracle-identity-column/
If we want to be able to override default generated value(identity insert) we have to use `BY DEFAULT`